code/game/objects/items/toys.dm

```diff
@@ -438,16 +437,6 @@
 	item_state = "egg3" // It's the green egg in items_left/righthand
 	item_color = "green"
 
-/obj/item/toddler
-	icon_state = "toddler"
-	name = "toddler"
-	desc = "This baby looks almost real. Wait, did it just burp?"
-	force = 5
-	w_class = WEIGHT_CLASS_BULKY
-	slot_flags = SLOT_FLAG_BACK
-
-
-//This should really be somewhere else but I don't know where. w/e
```

Comment:
This comment belongs to the inflatable duck?

Reply:
Ah, ok. They were right next to each other so I thought it was a note for toddlers... since toddlers aren't toys. 🧐 I can fix this when I get home.

Reply:
This is put back now